### DISCUSSION TITLE
2 instances of chatterino on startup

### DISCUSSION THREAD

**Discussion by Sempiternalvr** [Chatterino/chatterino2#5307] [Q&A] [answered]
is this normal? or is something pretending to be chatterino
![image](https://github.com/Chatterino/chatterino2/assets/166378594/99c8259f-d8bc-41c1-9e54-39b2de5fc9df)

**Comment by Felanbird** [comment]
It might be our crashpad handler for crash logs, does it happen to look like this on Task Manager?
![image](https://github.com/Chatterino/chatterino2/assets/41973452/8e2bb17f-7879-453d-a19f-f6020a565965)

**Reply by Felanbird** [reply]
Actually, inside the `startup` tab of `Task Manager` it should say the path for both of those chatterino's

**Reply by Nerixyz** [reply]
Crashpad shouldn't register itself on startup (though it might be shown in Taskmanager [?]).

**Comment by Nerixyz** [comment]
You should check on [AutoRuns](https://learn.microsoft.com/en-us/sysinternals/downloads/autoruns) if there are actually two instances and their paths.

**Reply by Sempiternalvr** [reply]
just checked autorun, only one instance up and this is what it looks like in task manager, should it be fine?
![image_2024-04-08_052127871](https://github.com/Chatterino/chatterino2/assets/166378594/dc467195-e040-4c19-ad6e-20e77edf501f)

**Reply by Nerixyz** [reply, marked as answer]
> just checked autorun, only one instance up and this is what it looks like in task manager, should it be fine?
> ![image_2024-04-08_052127871](https://github.com/Chatterino/chatterino2/assets/166378594/dc467195-e040-4c19-ad6e-20e77edf501f)
> 

Yea that should be correct. Sometimes Taskmanager doesn't associate the crashpad-handler with chatterino for some reason.
